ui/src/app/shared/header/chips/enum-selection.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core";
+
+@Component({
+  selector: 'jm-enum-selection',
+  templateUrl: './enum-selection.component.html',
+})
+export class EnumSelectionComponent implements OnInit {
+  @Input() initialChipValue: string;
+  @Input() chipKey: string;
+  @Input() chipOptions: string[];
+  @Output() updateValue: EventEmitter<string> = new EventEmitter();
+
+  currentChipValue: string;
+
+  ngOnInit(): void {
+    if (this.initialChipValue != null) {
```

Comment:
Is this check really needed? Isn't the `currentChipValue` just going to be `undefined` anyways?

Reply:
It's not undefined if there is a relevant parameter in the query string (e.g. `?q=flag%3Dfollow-up`)